test/regress/run_regression.py

```diff
@@ -473,7 +482,7 @@ def main():
 
     timeout = float(os.getenv('TEST_TIMEOUT', 600.0))
 
-    return run_regression(args.enable_proof, args.with_lfsc, args.dump,
+    return run_regression(args.enable_proof, args.enable_proof, args.dump,
```

Comment:
I think that it would be good to remove the separate argument for unsat cores from `run_regression()` and the `--with-lfsc` from the argument parser above.